### ISSUE TITLE
The element to open the widget does not have an alternative text

### ISSUE THREAD

**Issue by sharkboyto** [Jerit-Baiju/a11y-widget#2]
Using the widget with the screen reader, I noticed that the clickable element that opens the widget does not have alternative text. In addition, it is not an element that can be reached with the tab key, so it does not receive focus with the keyboard. Therefore, it should be converted into a link or a form button.
